### PR TITLE
Devise set up + home controller

### PR DESCRIPTION
**Pull request by yucombinator**
Making login great again. :ship:

### CODE REVIEW

**Comment by Deon** [comment]
`.gitignore` all `/.idea` files please.

File: Gemfile [hunk 1]
```diff
@@ -44,3 +43,6 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+# Auth!
+gem 'devise', :github => 'plataformatec/devise', :branch => 'master'
```
Comment (Deon) [inline Gemfile:50]: Leave a comment w/ this, re: why we're doing it this way.

File: app/views/home/index.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+Dank tanque
```
Comment (Deon) [inline app/views/home/index.erb:1]: Danque Tanque, aha I kid.

Comment (yucombinator) [inline app/views/home/index.erb:1]: #wontfix
#dankForLife

Comment (Deon) [inline app/views/home/index.erb:1]: :disappointed:

**Comment by Deon** [comment]
:+1:

**Pull request merged**